catalog/nsili/catalog-nsili-transformer/src/main/java/com/connexta/alliance/nsili/transformer/DAGConverter.java

```diff
@@ -1171,9 +1171,11 @@ public static void logMetacard(Metacard metacard, String id) {
             if (attribute != null) {
                 if (attribute.getValues() != null) {
                     String valueStr = getValueString(attribute.getValues());
-                    LOGGER.trace("{} :  {} : {}", id, descriptor.getName(), valueStr);
+                    LOGGER.trace("{} :  {}", id, descriptor.getName() + " : " +
```

Comment:
Don't have to concatenate  :  `LOGGER.trace("{} :  {} : {}", id, descriptor.getName(), valueStr);